### PR TITLE
added docs for Docker Cloud and Datacenter.

### PR DESCRIPTION
**Pull request by scriptnull**
https://github.com/Shippable/docsv2/issues/302

### CODE REVIEW

File: sources/pipelines_overview.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ page_keywords: Deploy multi containers, microservices, Continuous Integration, C
 
 ##Overview
 
-In addition to Continuous Integration, Shippable gives you an easy way to provision, configure, and deploy Dockerized applications to  popular container services like Amazon's EC2 Container Service (ECS) and Google Container Engine (GKE). Support for Microsoft Azure and Red Hat Openshift 3 is in the pipeline and planned for a April 2016 release.
+In addition to Continuous Integration, Shippable gives you an easy way to provision, configure, and deploy Dockerized applications to  popular container services like Amazon's EC2 Container Service (ECS), Google Container Engine (GKE), Docker Cloud and Docker Datacenter. Support for Microsoft Azure and Red Hat Openshift 3 is in the pipeline and planned for a April 2016 release.
```
Comment (abhijitkini) [inline sources/pipelines_overview.md]: Change April to Q2

**Comment by abhijitkini** [comment]
@scriptnull there's one more doc that needs to be written & updated.
How to use Docker Cloud & Docker Datacenter with Shippable. Just the basics. You can see examples on http://docs.shippable.com/ht_overview/

I have added one comment on a particular line.

Rest of it looks great.

**Comment by scriptnull** [comment]
Freezing this PR to get it reviewed completely and merged. 
Once we merge this PR, I will be complete the How-to tutorial for Docker Cloud and add the links for those in `ht_overview` page and send a new PR. It would be easier this way.

File: sources/ht_docker_datacenter.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+page_title: Deploying to Docker Datacenter
+page_description: How to deploy your application to Docker Datacenter
+page_keywords: docker datacenter, continuous deployment, CI/CD
+
+## Docker Datacenter
+Docker Datacenter CaaS (Containers-as-a-Service) enables enterprises to deploy Containers on-premise and on their virtual private cloud (VPC). We will see Shippable CI/CD workflow, that involves the following components of Docker Datacenter.
```
Comment (abhijitkini) [inline sources/ht_docker_datacenter.md]: on-premises is the correct usage.

File: sources/ht_docker_datacenter.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+- __Docker Trusted Registry__ (DTR) - Pull and push Images to your Docker Datacenter.
```
Comment (abhijitkini) [inline sources/ht_docker_datacenter.md]: use images instead of Images

File: sources/ht_docker_datacenter.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+- __Universal Control Plane__ (UCP) - Deploy and manage containers on your Docker Datacenter.
+
+### Continuous Integration
+Throughout the demo, We will be using a [sample node.js application](https://github.com/shippable/sample_node_ddc) to demonstrate the workflow.
+
```
Comment (abhijitkini) [inline sources/ht_docker_datacenter.md]: Throughout the demo, a sample node.js application is used to demonstrate the workflow.

File: sources/ht_docker_datacenter.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+To enable your project to use Shippable CI, it should be having a `shippable.yml` file that has instructions run your tests and push your docker images to a registry. To know more about the CI workflow, please check the [CI overview](ci_overview).
+
```
Comment (abhijitkini) [inline sources/ht_docker_datacenter.md]: Once you enable your project, it is critical to have a shippable.yml file to use Shippable CI. The shippable.yml file includes instructions to run your tests......

File: sources/ht_docker_datacenter.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+Lets take the `shippable.yml` file of the sample application.
```
Comment (abhijitkini) [inline sources/ht_docker_datacenter.md]: Let's review the shippable.yml.....

File: sources/ht_docker_datacenter.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+  post_ci:
+    - docker build -t ship-dtr.in/shippable-ci/sample_node_ddc:$BRANCH.$BUILD_NUMBER .
+    - docker push ship-dtr.in/shippable-ci/sample_node_ddc:$BRANCH.$BUILD_NUMBER
+
+integrations:
+  hub:
+    - integrationName : "Shippable-DTR"
+      type : Docker Trusted Registry
+```
+In this `ci` section, we issue commands to run our tests and once the tests complete, we will be pushing the docker image to a registry. ( Check the [supported registries](gs_supported.md) ).
```
Comment (abhijitkini) [inline sources/ht_docker_datacenter.md]: In the ci section shown above, we issue commands to run tests. Once the tests are complete, the docker image is pushed to a registry.

File: sources/ht_docker_datacenter.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+In our sample application, we will be pushing the docker images to Docker Trusted Registry using the integration named `Shippable-DTR` ( Check [how to create integration for a docker registry in Shippable](int_docker_registries.md))
```
Comment (abhijitkini) [inline sources/ht_docker_datacenter.md]: In the sample application, the docker images are pushed to the Docker Trusted Registry......

File: sources/ht_docker_datacenter.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+
+### Continuous Delivery
+After we push our images to Docker Trusted Registry, we will automatically deploy the latest image to Docker Datacenter. Shippable Pipelines allows us to deploy containers to Docker Datacenter both manually and automatically, whenever the project build finishes.
```
Comment (abhijitkini) [inline sources/ht_docker_datacenter.md]: After the images are pushed to Docker Trusted Registry, the latest image is automatically deployed to Docker Datacenter. Shippable Pipelines provides the choice of deploying containers to Docker Datacenter, manually & automatically, once the project build is complete.

File: sources/ht_docker_datacenter.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+
+Configuring your deployment process involves two steps.
```
Comment (abhijitkini) [inline sources/ht_docker_datacenter.md]: replace "your" with "the"

File: sources/ht_docker_datacenter.md [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+* __Environment Creation__ - Specify the infrastructure to be used to deploy ( in our case, it is the default swarm cluster of  Universal Control Plane )
+* __Pipeline Creation__ - Specify the image, tag pattern and other configurations like port mapping, environment variables etc. for deployment.
+
+#### Environment Creation
+You can follow the instructions [here](pipelines_configure#using-an-existing-docker-datacenter) to create a new Environment.
+<img src="../images/ddc_environment_creation.png" alt="Environment Creation" style="width:800px;"/>
+
+#### Pipeline Creation
+After Creating an Environment, Click on "Add Pipeline" you will be redirected to Pipeline creation page.
+
```
Comment (abhijitkini) [inline sources/ht_docker_datacenter.md]: click on 'Add Pipeline' and you will be redirected....

**Comment by abhijitkini** [comment]
I've recommended few changes.

**Comment by scriptnull** [comment]
@abhijitkini Thanks. I will update the PR and tag you back.

**Pull request merged**